src/main/scala/io/iohk/ethereum/network/PeerManagerActor.scala

```diff
@@ -35,6 +35,7 @@ class PeerManagerActor(
     with ActorLogging
     with Stash
     with BlacklistSupport {
+  override val maxBlacklistedNodes: Int = 5000
```

Comment:
Why do we need to override this value? Couldn't we change it in the base trait? Or maybe base trait shouldn't use any default value?

Reply:
we override it as other components do not need such large value, as usually they do not see more than few peers. PeerManageActor se more peers as it is almost on the lowset level of the stack, so it will see ~5k peers from which most will be useless (not neceserilly malicious, but just not usefull).

This whole blacklisting/marking useless peers approach sucks, but here I just wanted to tweak some values to more realitics one.

One thing we could easliy imporoved is to use here value from `DiscoveryConfig.nodesLimit` as we should not have more blacklisted peers than those found in discovery. wdyt ?

Reply:
I agree with you. We could use `nodesLimit` here. Please add a comment why such value (as you explain above)